lib/web/fetch/webidl.js

```diff
@@ -216,6 +216,21 @@ webidl.util.IntegerPart = function (n) {
   return r
 }
 
+webidl.util.Stringify = function (V) {
+  const type = webidl.util.Type(V)
+
+  switch (type) {
+    case 'Symbol':
+      return `Symbol(${V.description})`
+    case 'Object':
+      return JSON.stringify(V)
+    case 'String':
+      return V.length ? V : '""'
```

Comment:
So if V has a Value it wont be having double quotes but if it is empty it has double quotes. Not that nice.